datafeeds/parsers/json/json_zebra_motion_parser.py

```diff
@@ -0,0 +1,100 @@
+import json
+import re
+from datafeeds.parser_base import ParserInputException, ParserBase
+
+
+class JSONZebraMotionWorksParser(ParserBase):
+    @classmethod
+    def parse(cls, zebra_motion_json):
+        """
+        Parse JSON that contains Zebra MotionWorks data
+        Format is as follows:
+        [{
+            key: '2020casj_qm1',
+            times: [<elapsed time (float seconds)>, 0.0, 0.5, 1.0, 1.5, ...],
```

Comment:
Just confirming - the times I saw in the sample Zebra data on Chief looked to be absolute times, not relative times. Are these going to be absolute or relative?

Reply:
We're talking to them, and they will be relative.